.env_SAMPLE

```diff
@@ -108,9 +122,9 @@ export WORDPRESS=http://www.consumerfinance.gov
 # TESTING (optional) - for handling testing of the site.
 ########################################################
 
-#export HTTP_HOST=localhost
-#export HTTP_PORT=8000
-#export SELENIUM_URL=http://localhost:4444/wd/hub
+#export SITE_DESC=cfgov
+#export TEST_HTTP_HOST=localhost
+#export TEST_HTTP_PORT=8000
```

Comment:
Thanks, that's a helpful explanation. It feels like this is getting somewhat complicated, when all we need is just a way to set a bunch of environment variables. Did you specifically want to change the behavior so that you don't have to do anything special to `.env` in order to run the tests? Right now the comment in `.env_SAMPLE` tells you to uncomment those test variables, but `TEST.md` doesn't mention them.

What if we left `.env` as it was (commented out test variables, with defaults), made `TEST.md` more explicit about the need to set those, and then added a check in `environment.js` that actually fails if any of those variables are unset? That way you could still easily modify `.env` to override defaults as needed.